styles/simplified-styles.css

```diff
@@ -95,6 +95,35 @@ em > a:hover::after {
     padding-bottom: 0
 }
 
+.text-block a {
```

Comment:
It seems there's a similar styling effect present at https://github.com/hlxsites/zeiss/blob/12ff93f2ede1e2007f18fbac20bb522519b38933/styles/simplified-styles.css#L50-L76 for `em > a` and can be seen on https://main--zeiss--hlxsites.hlx.page/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020 for the same `Further Information` section. 

So probably we can have a common style and avoid the repetition.

Reply:
Good catch, thank you @sdmcraft ! I have pushed commit which merges the styles.